platform/security/sts/security-sts-ldapclaimshandler/src/main/java/ddf/security/sts/claimsHandler/LdapClaimsHandler.java

```diff
@@ -112,8 +116,34 @@ public ProcessedClaimCollection retrieveClaimValues(ClaimCollection claims,
 
             connection = connectionFactory.getConnection();
             if (connection != null) {
-                BindResult bindResult = connection.bind(bindUserDN,
-                        bindUserCredentials.toCharArray());
+                BindRequest request;
```

Comment:
Regardless, it would be best if the duplicate code blocks here and in `RoleClaimsHandler` (and if possible) `SslLdapLoginModule` could be extracted out to a common utility method.